src/components/Footer.tsx

```diff
@@ -43,7 +43,23 @@ export const Footer: React.FC = () => (
     </Box>
     <Flex>
       <Spacer />
-      <Text fontSize="xs">v{process.env.REACT_APP_VERSION || "???"}</Text>
+      <Box fontSize="xs">
+        <Link
+          href={
+            process.env.REACT_APP_SEMVER
+              ? `https://github.com/labeleven-dev/bettercallsol/releases/tag/v${process.env.REACT_APP_SEMVER}`
```

Comment:
Looks like a broken link here?

Reply:
On the preview site? We don't version previews so it's hardcoded to `PREVIEW` in the pipeline. And there is no `vPREVIEW` release tag.